types/params.go

```diff
@@ -75,6 +75,13 @@ type VersionParams struct {
 	AppVersion uint64 `json:"app_version"`
 }
 
+// TimestampParams influence the validity of block timestamps.
+type TimestampParams struct {
```

Comment:
Do we want to add a default?

Reply:
Set some basics and will follow up in https://github.com/tendermint/tendermint/issues/7202

Reply:
These would be Synchronous or Timing parameters, wouldn't them?